src/plugins/ecs/__init__.py

```diff
@@ -74,7 +73,52 @@ def main(text):
 				ret = ret + cluster.split('/')[-1] + '\n'
 
 			return ret
-		elif 'services' in text:
+		
+		elif tasks_check_text(text):
+
+			tasks_lookup_term = tasks_get_lookup_term(text)
+			fields = []
+			attachments = []
+
+			if len(text) == 0:
+				return "I need a cluster name to complete the requested operation. To view the cluster names, use 'jarvis ecs list clusters <region>'"
+
+			if 'running' in text:
+				text.remove("running")
+
+				try:
+					resulting_array = get_task_list(instance=None, next_token=None, cluster=text[0], ecs=ecs)
+					query_result = ecs.describe_tasks(cluster=text[0], tasks=resulting_array)
+					instance_task_families = parse_tasks(query_result['tasks'], tasks_lookup_term)
+
+					if len(instance_task_families) == 0:
+						return "No tasks where found matching the lookup term for tasks. To look up a particular task, use 'jarvis ecs list tasks-><optional term> running <cluster> [in <region/account>]' "
+
+					for tasks in instance_task_families:
+						fields.append({
+								'title': tasks,
+								'value': 'Count: ' + str(instance_task_families[tasks]['count']),
+								'short': True
+							})
+
+					attachments.append({
+						'fallback': 'Service List',
+						'title': 'List of Running Tasks',
+						'fields': fields
+					})	
+
+					return attachments
+
+
+				except Exception as e:
+					print e
+					return "Cluster " + text[0] + " was not found in region " + region
+
+			else:
+				return "No valid option for command jarvis ecs list tasks found. Please review /jarvis --help and try again."
+
+
+        elif 'services' in text:
```

Comment:
It looks like you removed the 'list services' option - this elif is at a different indentation level than it used to be...